src/Passes/Source/Apps/Qat/Config.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    /// Main configuration class for the qat command-line program.
+    class QatConfig
```

Comment:
I would suggest to unify the class name 
`QatConfig` and the file name 
`Config.hpp`.